inbm/fpm/mqtt/template/usr/bin/mqtt-ensure-keys-generated

```diff
@@ -63,8 +63,8 @@ fix_permissions() {
         if [ "$GROUP" != "lost+found" ] ; then
             check_no_insecure_user "$USER"
             
-            if [ "$ID" != "tiber" ]; then
-                # Only add groups and users if not on 'tiber'
+            if [ "$ID" != "Edge Microvisor Toolkit" ]; then
```

Comment:
Does this match `ID=...` in `/etc/os-release`? Usually `ID` is a short string.

Reply:
Yes. The new image shows the ID="Edge Microvisor Toolkit"